satpy/FY4A_agri_introduction(EN).ipynb

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
<p>I don't see the colormaps being used in this tutorial, so maybe they can be removed ?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Yes, I will remove them. I'll use them in another tutorial :)</p>